checks/raw/security_policy_test.go

```diff
@@ -138,7 +138,7 @@ func TestSecurityPolicy(t *testing.T) {
 			mockRepo := mockrepo.NewMockRepo(ctrl)
 
 			mockRepoClient.EXPECT().ListFiles(gomock.Any()).Return(tt.files, nil).AnyTimes()
-			mockRepo.EXPECT().Org().Return(nil).AnyTimes()
+			mockRepo.EXPECT().Org().Return("").AnyTimes()
```

Comment:
Curious why this change was needed?

Reply:
This was initially because the return type of `Org()` was changed from RepoClient to string (the latter is not nilable)